[Question] : new SDK style creates sys.trace_xe_action_map table

Can we avoid creating sys.trace_xe_action_map table 

```
2024-07-10T21:25:57.1337100Z Changed database context to 'CustomerScoring'.
2024-07-10T21:25:57.1421710Z PRE-DEPLOYMENT START TIME: Jul 10 2024  5:25:57:137PM
2024-07-10T21:25:57.1426869Z Current version is: 2428.00
2024-07-10T21:25:57.1430601Z BEGIN PRE-DEPLOYMENT SCRIPTS
2024-07-10T21:25:57.1432500Z END PRE-DEPLOYMENT SCRIPTS
2024-07-10T21:25:57.1432889Z FINISHED PROCESSING OF PRE-DEPLOYMENT INCREMENTAL SCRIPTS in 0.004s
2024-07-10T21:25:57.1433553Z PRE-DEPLOYMENT END TIME: Jul 10 2024  5:25:57:140PM
2024-07-10T21:25:57.1445347Z Creating Table [sys].[trace_xe_action_map]...
2024-07-10T21:25:57.1756032Z Msg 2760, Level 16, State 1, Server EC2AMAZ-4I6NHUO, Line 1
2024-07-10T21:25:57.1756685Z The specified schema name "sys" either does not exist or you do not have permission to use it.
2024-07-10T21:25:57.2664555Z ##[error]Cmd.exe exited with code '1'.
```

--- a/README.md
+++ b/README.md
@@ -234,12 +234,12 @@ It will assume that the `.dacpac` file is inside the `tools` folder of the refer
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="MyDatabasePackage" Version="1.1.0" DacpacName="SomeOtherDatabase" />
+        <PackageReference Include="MyDatabasePackage" Version="1.1.0" DacpacName="SomeOtherDacpac" />
     </ItemGroup>
 </Project>
 ```
 
-This will add a reference to the `tools\SomeOtherDatabase.dacpac` file inside the `MyDatabasePackage` package. Note that if that file doesn't exist within the package, the package reference will still be silently ignored. However the build will most likely fail if your project actually references objects from the reference package.
+This will add a reference to the `tools\SomeOtherDacpac.dacpac` file inside the `MyDatabasePackage` package. Note that if that file doesn't exist within the package, the package reference will still be silently ignored. However the build will most likely fail if your project actually references objects from the reference package.
 
 By default the package reference is treated as being part of the same database. For example, if the reference package contains a `.dacpac` that has a table and a stored procedure and you would `dotnet publish` the project the table and stored procedure from that package will be deployed along with the contents of your project to the same database. If this is not desired, you can add the `DatabaseVariableLiteralValue` item metadata to the `PackageReference` specifying a different database name:
 
@@ -309,12 +309,12 @@ Microsoft has recently released NuGet packages containing the definitions of the
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.SqlServer.Dacpacs.Master" Version="160.0.0" DacpacName="master" />
+        <PackageReference Include="Microsoft.SqlServer.Dacpacs.Master" Version="160.2.1" DacpacName="master" />
     </ItemGroup>
 </Project>
 ```
 
-The above example references the `master` database from the [Microsoft.SqlServer.Dacpacs.Master](https://www.nuget.org/packages/Microsoft.SqlServer.Dacpacs.Master) NuGet package. Please note that there are different versions of that package for different versions of SQL Server. It is recommended to reference the same version of the package as the `SqlServerVersion` you are targeting with your project, as seen in the example above.
+The above example references the `master` database from the [Microsoft.SqlServer.Dacpacs.Master](https://www.nuget.org/packages/Microsoft.SqlServer.Dacpacs.Master) NuGet package. Please note that there are different major versions of that package for different versions of SQL Server. It is recommended to reference the most recent minor/patch version of the package as the `SqlServerVersion` you are targeting with your project, as seen in the example above.
 
 For other variants of SQL Server / Azure SQL Database there are dedicated packages as [listed here](https://www.nuget.org/packages/Microsoft.SqlServer.Dacpacs.Master#readme-body-tab).
 

